externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td

```diff
@@ -0,0 +1,29 @@
+#ifndef TORCH_MLIR_DIALECT_TCP_BASE
+#define TORCH_MLIR_DIALECT_TCP_BASE
+
+include "mlir/IR/OpBase.td"
+
+def Tcp_Dialect : Dialect {
+  let name = "tcp";
+  let cppNamespace = "::mlir::torch::tcp";
+  let description = [{
+    Tensor Compute Primitives (TCP) dialect.
+
+    TCP is a mid-level dialect in MLIR that enables transformations that
+    complement those in existing dialects, by supporting ops at a
+    high-level of abstraction.
+
+    Status: This is being boostrapped under Torch-MLIR Dialects.
```

Comment:
Is this necessary here?  This is more about the roadmap than the technical content of the dialect right?

Reply:
Removed.